### DISCUSSION TITLE
中文逗号和模板不同

### DISCUSSION THREAD

**Discussion by Nono-404** [note286/xduts#108] [Q&A] [answered]
在latex编译器中输入中文逗号，
我的：![image](https://user-images.githubusercontent.com/56815855/228165840-e1f17714-25b3-4c92-b697-df5584c23991.png)
模板的逗号：
![image](https://user-images.githubusercontent.com/56815855/228166166-1960b88c-6cbd-4000-a89c-dbdd3c0f8c21.png)
   这个是我没设置好吗

**Comment by note286** [comment, marked as answer]
中文字体默认为`fandol`，你可以改为你熟悉的`win`。

**Comment by sikouhjw** [comment]
输入的是逗号，但在编辑器上显示的，是跟你编辑器的字体有关。

同理，输出也是。
